akka-typed/src/main/java/akka/typed/javadsl/ActorContext.java

```diff
@@ -107,6 +107,14 @@
   public <U> void watch(ActorRef<U> other);
 
   /**
+   * Register for termination notification with a custom message once the Actor identified by the
+   * given {@link akka.typed.ActorRef} terminates. This notification is also generated when the
+   * {@link akka.typed.ActorSystem} to which the referenced Actor belongs is declared as failed
+   * (e.g. in reaction to being unreachable).
```

Comment:
unreachable is the wrong word, because Cluster unreachable doesn't trigger Terminated. Terminated is triggered when Cluster member is removed from membership.